### PR TITLE
Update CheckHealthResponse

### PR DESCRIPTION
**Pull request by bigsheeper**
Signed-off-by: bigsheeper <yihao.dai@zilliz.com>

issue: #71

### CODE REVIEW

File: proto/milvus.proto [hunk 1]
```diff
@@ -1374,11 +1374,20 @@ message GetVersionResponse {
...
 message CheckHealthRequest {
 }
 
 message CheckHealthResponse {
   common.Status status = 1;
   bool isHealthy = 2;
-  repeated string reasons = 3;
-}
\ No newline at end of file
+  repeated QuotaState quota_states = 3;
```
Comment (xiaofan-luan) [inline proto/milvus.proto:1391]: should not change the order to proto

**Comment by bigsheeper** [comment]
issue: https://github.com/milvus-io/milvus/issues/21387

**Comment by bigsheeper** [comment]
issue: #71

**Pull request merged**